.github/workflows/appimage.yml

```diff
@@ -0,0 +1,124 @@
+# Manually triggered build of mlnet, mlgui and utils binaries and appimage with mlnet+gui
+
+name: Build AppImage
+
+on:
+  workflow_dispatch
+
+env:
+  APPIMAGEBUILDER_VERSION: 1.1.0
+  APPIMAGEBUILDER_SHA1SUM: 6f83a789f6c47a745b97d1b0b3f1df2e7eea7a09
+
+jobs:
+  build:
```

Comment:
i think ideally this build part of pipeline should be move to main pipeline, and then here keep only appimage (and use artifacts produced by main build pipeline)